users/serializers.py

```diff
@@ -32,3 +32,9 @@ def create(self, validated_data):
         user = User.objects.create_user(**validated_data)
 
         return user
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = User
+        fields = ['id', 'username', 'email', 'first_name', 'last_name']
```

Comment:
Hey mate, we don't need first_name, last_name in the listing of users. We only need to send username and respective email through API. Why to make API bulgy?

Reply:
Okay, I will make this changes and resubmit new pr so that you can review it again.